qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/TemplateCodeActionFactory.java

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.services.diagnostics;
+
+import java.util.Collections;
+
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.CodeActionKind;
+import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.Diagnostic;
+
+import com.redhat.qute.ls.commons.client.ConfigurationItemEdit;
+import com.redhat.qute.ls.commons.client.ConfigurationItemEditType;
+import com.redhat.qute.ls.commons.client.QuteClientCommandConstants;
+
+/**
+ * Specific code action factory for Qute.
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class TemplateCodeActionFactory {
+
+	// Enable/Disable Qute validation
+
+	private static final String UNASSIGNED_EXCLUDED_SECTION = "quarkus.tools.qute.validation.enabled";
+
+	private static final String UNASSIGNED_EXCLUDE_CODE_ACTION_TITLE = "Do you want to disable Qute validation?";

```

Comment:
Disable Qute validation

Not a question

Reply:
fixed